src/components/pages/datasets/DatasetsList.jsx

```diff
@@ -82,11 +83,17 @@ const DatasetsList = ({ match, history }) => {
                     {i18n.language === 'en' ? item.name_eng : item.name}
                   </td>
                   <td className="text-center">
-                    {dateFormat(item.source_last_update)}
+                    {dateFormat(item.updated_at)}
+                  </td>
+                  <td className="text-center">
+                    {t(item.status)}
```

Comment:
Localization of status must be implemented on backend. Return to frontend visible value.